recipes/colander/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "colander" %}
+{% set version = "1.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/colander-{{ version }}.tar.gz
+  sha256: 259592a0d6a89cbe63c0c5771f9c0c2522387415af8d715f599583eac659f7d4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - enum34
+    - iso8601
+    - setuptools >=11.3
+    - translationstring
+
+test:
+  imports:
+    - colander
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Pylons/colander
+  summary: A simple schema-based serialization and deserialization library
+  license: Mup
```

Comment:
It's similar but I don't think it's actually `Mup`: https://spdx.org/licenses/Mup.html

Reply:
They say BSD-derived: https://github.com/Pylons/colander/blob/master/setup.cfg#L8

Looks like it is this one: https://fedoraproject.org/wiki/Licensing:BSD#Modification_Variant

But I could not find a SPDX form for it.

Reply:
Yep. Looks like it. Thanks!